documentation/developer-guide/modules/tooling-guide/pages/samm-cli.adoc

```diff
@@ -5,10 +5,16 @@
 
 The SAMM CLI is a command line tool for the validation of Aspect Models and the generation of artifacts, such as documentation or code, from Aspect Models.
 
-TIP: Download latest version: icon:download[] https://github.com/eclipse-esmf/esmf-sdk/releases/download/v{esmf-sdk-version}/samm-cli-{esmf-sdk-version}.jar[samm-cli-{esmf-sdk-version}.jar]
+The SAMM CLI offers multiple deployment options (you can download it bellow):
+
+TIP: Download latest version executable jar: icon:download[] https://github.com/eclipse-esmf/esmf-sdk/releases/download/v{esmf-sdk-version}/samm-cli-{esmf-sdk-version}-x86_64.jar[samm-cli-{esmf-sdk-version}-x86_64.jar]
```

Comment:
Download executable jar (latest):

- @atextor , @Yauhenikapl , I propose to use "native executable" instead of "native binary". Graal themselves use the wording (https://www.graalvm.org/latest/reference-manual/native-image/) and for "the basic windows user", "executable" carries "I can execute it" while "binary" can lead to ...oO(???)